specification/synapse/data-plane/readme.md

```diff
@@ -65,6 +65,18 @@ input-file:
 - Microsoft.Synapse/preview/2019-06-01-preview/artifacts.json
 ```
 
+### Tag: package-arm-artifacts-2020-01-01-preview
+
+These settings apply only when `--tag=package-access-control-2020-01-01-preview` is specified on the command line.
```

Comment:
>package-access-control-2020-01-01-preview [](start = 38, length = 41)

Let's use `package-arm-artifacts-2020-01-01-preview`. I need to confirm with PM side about the API version string and package name